plugins/linux-vaapi/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+project(linux-vaapi)
+
+find_package(LibVa)
+find_package(X11 REQUIRED)
+
```

Comment:
To allow graceful failure/omission of this module if libva or other dependencies are not found on the system, add this here:

```
if(NOT LIBVA_INCLUDES_FOUND AND ENABLE_VA)
	message(FATAL_ERROR "LibVA includes not found but set as enabled")
elseif(NOT LIBVA_INCLUDES_FOUND)
	message(STATUS "LibVA includes not found, VAAPI plugin disabled")
	return()
endif()
```